### PR TITLE
integrate video stream approach to cvat

### PR DESCRIPTION
**Pull request by ygnn123**

### CODE REVIEW

File: cvat-core/build.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+npm run build -- --mode=development
```
Comment (nmanovic) [inline cvat-core/build.sh]: Please remove the file.

File: cvat-data/build.sh [hunk 2]
```diff
@@ -0,0 +1,2 @@
+npm run build -- --mode=development
```
Comment (nmanovic) [inline cvat-data/build.sh]: Please remove the file

File: cvat-core/build.sh (hunk 1 above)
Comment (ygnn123) [reply cvat-core/build.sh]: later... I do not want to copy files manually

Comment (nmanovic) [reply cvat-core/build.sh]: @ygnn123, you can keep them locally. No problems, just remove them from the pull request.

**Comment by nmanovic** [comment]
@ygnn123 , please look at TRAVIS build. It seems the PR has some problems.

**Comment by nmanovic** [comment]
@ygnn123 , the PR has conflicts. Could you please work with @bsekachev to resolve them and update the branch ASAP?

**Comment by nmanovic** [comment]
outdated PR.

**Pull request closed without merging**